### DISCUSSION TITLE
Q&A: Aspect Ratios window

### DISCUSSION THREAD

**Discussion by kalle07** [lllyasviel/Fooocus#3297] [Q&A] [answered]
where i can change the code that this window is open on startup?

![grafik](https://github.com/user-attachments/assets/6f914b63-53e5-4384-a92d-72f153d060c7)

**Comment by mashb1t** [comment, marked as answer]
Change `open=False` to True in https://github.com/lllyasviel/Fooocus/blob/main/webui.py#L549
